Nautilus/Patchers/PDAEncyclopediaTabPatcher.cs

```diff
@@ -0,0 +1,19 @@
+using HarmonyLib;
+using Nautilus.Handlers;
+
+namespace Nautilus.Patchers;
+
+internal class PDAEncyclopediaTabPatcher
+{
+    internal static void Patch(Harmony harmony)
+    {
+        harmony.PatchAll(typeof(PDAEncyclopediaTabPatcher));
+    }
+    [HarmonyPatch(typeof(uGUI_EncyclopediaTab))]
+    [HarmonyPatch(nameof(uGUI_EncyclopediaTab.Awake))]
+    [HarmonyPostfix]
+    internal static void EncyTabAwakePostfix(uGUI_EncyclopediaTab __instance)
+    {
+      ModDatabankHandler.Initialize(__instance);
```

Comment:
Compiler error... you do have an IDE right?

Reply:
yeah, I was just doing this from github file edits since I wasn't at home.

Reply:
fixed.